js/apps/admin-ui/src/identity-providers/IdentityProvidersSection.tsx

```diff
@@ -92,18 +93,31 @@ export default function IdentityProvidersSection() {
 
   useFetch(
     async () => {
-      const provider = await adminClient.realms.findOne({ realm });
-      if (!provider) {
-        throw new Error(t("notFound"));
+      const providers = await adminClient.identityProviders.find();
+      if (!providers) {
+        throw new Error(t("common:notFound"));
```

Comment:
We no longer namespace keys for translations.
```suggestion
        throw new Error(t("notFound"));
```